src/main/java/uk/gov/companieshouse/presenter/account/model/PresenterAccountAddress.java

```diff
@@ -51,11 +51,7 @@ private String validateAddressLine2(final String line) {
     }
 
     private String validatedTownOrCity(final String line) {
-        if (line == null || line.isBlank()) {
-            return "";
-        } else {
-            return getValidatedLine(line, COUNTY_MAX_LENGTH, "townOrCity failed validation");
-        }
+        return getValidatedLine(line, COUNTY_MAX_LENGTH, "townOrCity failed validation");
```

Comment:
Can we change the name of the length variable to town or city

Reply:
Yeah. I missed that.